engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/validator/SingleValueAnnotationValidatorExtractor.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.engine.definition.validator
+
+import java.lang.reflect.Parameter
+
+import javax.validation.constraints.{Max, Min}
+import pl.touk.nussknacker.engine.api.definition.{MaximalNumberValidator, MinimalNumberValidator, ParameterValidator}
+
+object SingleValueAnnotationValidatorExtractor extends ValidatorExtractor {
+  override def extract(params: ValidatorExtractorParameters): Option[ParameterValidator] = {
+    params.rawJavaParam match {
+      case param if param.getAnnotation(classOf[Min]) != null => extractMinimalNumberValidator(param)
```

Comment:
1. How it will work for params with both `@Min` and `@Max`? Please add a test for that
2. I think that we need something similar to  `AnnotationValidatorExtractor` but taking parameter and wonder if we can reuse it for min / max and for literal?

Reply:
Yes, I think we should make 
class AnnotationValidatorExtractor[T<:Annotation:ClassTag](prepare: T => ParameterValidator) 
or sth similar, and have helper method for current case (that is, annotation without params)

Reply:
Now it's ok, thanks